src/state/create-store.js

```diff
@@ -1,12 +1,14 @@
 // @flow
+/* eslint-disable no-underscore-dangle */
 import { applyMiddleware, createStore, compose } from 'redux';
 import thunk from 'redux-thunk';
 import reducer from './reducer';
 import hookMiddleware from './hook-middleware';
 import type { Store, Hooks } from '../types';
 
-// eslint-disable-next-line no-underscore-dangle
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const composeEnhancers = typeof window === 'object'
```

Comment:
Can you please add a comment above stating 
1. We are checking if window is available because it might not be when server side rendering
2. That this came from https://github.com/zalmoxisus/redux-devtools-extension#12-advanced-store-setup